### PR TITLE
Add support for review events

### PR DESCRIPTION
**Pull request by alexrudd**
Hey, this PR adds support for the [review_requested](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/issue-event-types#review_requested) and [review_request_removed](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/issue-event-types#review_request_removed) issue events.

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @alexrudd !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
Thank you, @alexrudd 
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
